src/com/pty4j/unix/NativePtyExecutor.java

```diff
@@ -102,7 +103,7 @@ int exec_pty(String full_path, String[] argv, String[] envp, String dirpath, Str
    * @see <a href="https://man7.org/linux/man-pages/man2/ioctl_tty.2.html>ioctl_tty</a>
    */
   public static class WinSizeStructure extends Structure {
-    private static final List<String> FIELD_ORDER = ImmutableList.of("ws_row", "ws_col", "ws_xpixel", "ws_ypixel");
+    private static final List<String> FIELD_ORDER = Collections.unmodifiableList(Arrays.asList("ws_row", "ws_col", "ws_xpixel", "ws_ypixel"));
```

Comment:
Please use `List.of`

Reply:
Here and in other similar places in the PR.

Reply:
@segrey I am aware of List.of, but it was added in Java 8. I thought you wanted to keep supporting old Java versions.

Reply:
In fact, pty4j targets Java 11 (https://github.com/JetBrains/pty4j/blob/master/build.gradle#L58). This allows to implement some new java.lang.Process methods more efficiently.

Reply:
@segrey That's cool, will change them then.